pkg/web3/bindings/pow/pow.go

```diff
@@ -827,6 +827,7 @@ func (_Pow *PowTransactorSession) RenounceOwnership() (*types.Transaction, error
 //
 // Solidity: function submitWork(uint256 nonce, string nodeId) returns()
 func (_Pow *PowTransactor) SubmitWork(opts *bind.TransactOpts, nonce *big.Int, nodeId string) (*types.Transaction, error) {
+	opts.NoSend = true
```

Comment:
let's create a new submit work function here so we don't touch any of the old features. See my branch here https://github.com/Lilypad-Tech/lilypad/compare/feat/rpc-batch-ar?expand=1